lib/rubocop/cop/rspec/context_wording.rb

```diff
@@ -0,0 +1,61 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module RSpec
+      # `context` block descriptions should start with 'when', or 'with'.
+      #
+      # @see https://github.com/reachlocal/rspec-style-guide#context-descriptions
+      # @see http://www.betterspecs.org/#contexts
+      #
+      # Configurable using `Prefixes` option, defaults are 'when' and 'with'.
+      #
+      # @example
+      #   # bad
+      #   context 'the display name not present' do
+      #     ...
+      #   end
+      #
+      #   # good
+      #   context 'when the display name is not present' do
+      #     ...
+      #   end
+      class ContextWording < Cop
+        MSG = 'Start context description with %s.'.freeze
+        DEFAULT_PREFIXES = %w[when with].freeze
```

Comment:
I think this should also include `without` as it's the opposite context of `with`

Reply:
I think there is no need for default prefixes. Shouldn’t it work by defining prefixes in the configuration file?

Reply:
```
───────────────────────────────────────────────────
modified: config/default.yml
───────────────────────────────────────────────────
@ default.yml:49 @ RSpec/BeforeAfterAll:
RSpec/ContextWording:
  Description: "`context` block descriptions should start with 'when', or 'with'."
  Enabled: true
  Prefixes:
  - when
  - with
  StyleGuide: http://www.rubydoc.info/gems/rubocop-rspec/RuboCop/Cop/RSpec/ContextWording

RSpec/DescribeClass:
───────────────────────────────────────────────────
modified: lib/rubocop/cop/rspec/context_wording.rb
───────────────────────────────────────────────────
@ context_wording.rb:29 @ module RuboCop
      #   end
      class ContextWording < Cop
        MSG = 'Start context description with %s.'.freeze
        DEFAULT_PREFIXES = %w[when with].freeze

        def_node_matcher :context_wording, <<-PATTERN
          (block (send _ { :context :shared_context } $(str $_)) ...)
@ context_wording.rb:48 @ module RuboCop
        end

        def prefixes
          cop_config['Prefixes'] || DEFAULT_PREFIXES
          cop_config['Prefixes']
        end
```

ends up with:
```
     Failure/Error: prefixes.include?(description.split(' ', 2).first)

     NoMethodError:
       undefined method `include?' for nil:NilClass
```

@bquorning Is that correct to add those default options to `config/default.yml`?

If I stub default optionns in spec, how do we make sure those options are not removed from `config/default.yml` and break everything while specs still pass?